### PR TITLE
[Teamwork] Add `orderBy` query option when listing chats

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: api-reference/beta/api/chat-list.md [hunk 1]
```diff
@@ -661,6 +661,74 @@ Content-type: application/json
...
+#### Request
+
+Here is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "list_chats_orderby"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/beta/chats?orderBy=lastMessagePreview/createdDateTime desc
```
Comment (eddie-lee-msft) [inline api-reference/beta/api/chat-list.md:675]: missing $ before orderBy

Comment (eddie-lee-msft) [reply api-reference/beta/api/chat-list.md:675]: same in the response

**Review by eddie-lee-msft — APPROVED** [review]

File: api-reference/beta/api/chat-list.md (hunk 1 above)
Comment (JarbasHorst) [reply api-reference/beta/api/chat-list.md:675]: Added.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [17053](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17053) into this branch to include my final edits.

Hello @jessiechan27, I added one comment that might improve readability, if addressed.

**Comment by JarbasHorst** [comment]
@Lauragra, consider including [my changes](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17924) to this PR before merging it.

**Pull request merged**